requirements/edx/base.txt

```diff
@@ -156,3 +156,6 @@ analytics-python==0.4.4
 # Needed for mailchimp(mailing djangoapp)
 mailsnake==1.6.2
 jsonfield==1.0.3
+
+# Needed for credit notifications
+pynliner==0.5.2
```

Comment:
I believe @ahsan-ul-haq added this since some email clients strip the `<script>` tag from the message.  @ahsan-ul-haq could you confirm?

Most (all?) of our system emails don't currently have HTML styling.  Bulk email does have styling, but I believe the complete HTML including inline styles is currently stored in the database.

I agree that we could add more context to the comment.  @ahsan-ul-haq could you update the comment so it says something like:
```
Inlines CSS styles into HTML for email notifications.
```